doc/tutorials/duckdb-native-sqlalchemy.md

```diff
@@ -19,44 +19,58 @@ myst:
 
 # DuckDB (native vs SQLAlchemy)
 
-Beginning in 0.9, JupySQL supports DuckDB via a native connection and SQLAlchemy, both with comparable performance.
+Beginning in 0.9, JupySQL supports DuckDB via a native connection and SQLAlchemy, both with comparable performance. JupySQL has adds a small overhead; however, this overhead is constant.
 
 At the moment, the only difference is that some features are only available when using SQLAlchemy.
 
 +++
 
 ## Performance comparison (pandas)
 
-### DuckDB + SQLALchemy
-
 ```{code-cell} ipython3
 import pandas as pd
+import numpy as np
+
+num_rows = 1_000_000
+num_cols = 100
 
-df = pd.DataFrame({"x": range(1_000_000)})
+df = pd.DataFrame(np.random.randn(num_rows, num_cols))
 ```
 
+## Raw DuckDB
+
+```{code-cell} ipython3
+import duckdb
+conn = duckdb.connect()
+```
+
+```{code-cell} ipython3
+%%timeit
+conn.execute("SELECT * FROM df").df()
```

Comment:
Doubts:
1. Why do we need `.df()` / `.pl()` if `autopandas` / `autopolars` is set to True?
2. Is this notation documented somewhere?